test/es6/rlexe.xml

```diff
@@ -1070,7 +1070,7 @@
   <test>
     <default>
       <files>bug_OS_2553885.js</files>
-      <tags>exclude_win7,BugFix</tags>
+      <tags>exclude_win7,BugFix,exclude_xplat</tags>
```

Comment:
Are these test just missing an UTF8 BOM?
Would adding it fixes the test?
Is there a reason we can't add it?  (e.g. the test requires that it not present?)